src/item_action.cpp

```diff
@@ -307,9 +307,7 @@ void game::item_action_menu()
     const item_action_id action = std::get<0>( menu_items[kmenu.ret] );
     item *it = iactions[action];
 
-    if( u.invoke_item( it, action ) ) {
-        u.i_rem( it ); // Need to remove item
-    }
+    u.invoke_item(it, action);
```

Comment:
Should be styled like `u.invoke_item( it, action );` (with whitespaces inside brackets).

Reply:
Will fix my VS options so that i won't mess with style anymore.